careerfairsystems/arkad-app-2018
# Improve design of app

## Change description

Improves the design of the app.

## How to verify

Run the app on both iOS and Android and check the design. Make sure it complies with the design provided by @liljagustav.

## Issues fixed

This fixes #63.
